go/libkb/errors.go

```diff
@@ -1399,6 +1399,12 @@ func (e UnknownStreamError) Error() string {
 	return "unknown stream format"
 }
 
+type UnicodeUnsupportedError struct{}
+
+func (e UnicodeUnsupportedError) Error() string {
+	return "unicode not supported"
```

Comment:
I think we only allow extra whitespace in saltpack messages right now, and not other prefix characters, so we'd probably choke. A lot of sources recommend against prepending the UTF-8 BOM, for what it's worth: http://utf8everywhere.org/#faq.boms